src/__tests__/js/sources-util.js

```diff
@@ -48,33 +48,33 @@ describe('message duplication mitigation', () =>{
   // needed for repeated tls like '*explains meme' with other tls in between
   it('lets messages that are duplicates of a message outside of the last 10 seconds through', () => {
     const messages = [
-      message('Konichiwassup', 'Taishi', '00:10:00', AuthorType.mchad),
-      message('hello there', 'Taishi Ch.', '10:05',  0),
-      message('Konichiwassup', 'Taishi', '00:10:12', AuthorType.mchad),
-      message('hello there', 'Taishi Ch.', '10:18', 0),
-      message('Konichiwassup', 'Taishi Ch.', '10:30', 0),
-      message('hello there', 'Taishi', '00:10:40', AuthorType.mchad),
+      message('Konichiwassup', 'Taishi', 100000, AuthorType.mchad),
+      message('hello there', 'Taishi Ch.', 105000,  0),
+      message('Konichiwassup', 'Taishi', 112000, AuthorType.mchad),
+      message('hello there', 'Taishi Ch.', 118000, 0),
+      message('Konichiwassup', 'Taishi Ch.', 130000, 0),
+      message('hello there', 'Taishi', 140000, AuthorType.mchad),
     ];
     expect(removeDuplicateMessages(messages)).toEqual(messages);
   });
 
   it('lets non-duplicate messages of same source through', () => {
     const messages = [
-      message('Konichiwassup', 'Taishi', '00:10:00', 0),
-      message('hello there', 'Taishi Ch.', '10:05',  0),
-      message('good translation', 'Taishi', '00:10:12', 0),
-      message('another good translation', 'Taishi Ch.', '10:18', 0),
-      message('puhehehe', 'Taishi Ch.', '10:30', 0),
-      message('hehe', 'Taishi', '00:10:40', 0),
+      message('Konichiwassup', 'Taishi', 100000, 0),
+      message('hello there', 'Taishi Ch.', 105000,  0),
+      message('good translation', 'Taishi', 112000, 0),
+      message('another good translation', 'Taishi Ch.', 118000, 0),
+      message('puhehehe', 'Taishi Ch.', 130000, 0),
+      message('hehe', 'Taishi', 140000, 0),
     ];
     expect(removeDuplicateMessages(messages)).toEqual(messages);
   });
 
   it('admits duplicate messages of inside the last 10 seconds and from same source', () => {
     const messages = [
-      message('*Explains meme', 'Taishi', '00:10:00', AuthorType.mchad),
-      message('*Explains meme', 'Taishi Ch.', '10:05', 0),
-      message('*Explains meme', 'Taishi', '00:10:08', AuthorType.mchad),
+      message('*Explains meme', 'Taishi', 100000, AuthorType.mchad),
+      message('*Explains meme', 'Taishi Ch.', 105000, 0),
+      message('*Explains meme', 'Taishi', 108000, AuthorType.mchad),
     ];
     const expectedMessages = [
       messages[0],
```

Comment:
This test is still failing on local for some reason

Reply:
Jest is sussy, it works when you reclone the folder.

Reply:
Nevermind jest is weird